includes/modules/payment/securesubmit.php

```diff
@@ -219,7 +221,19 @@ public function before_process() {
 
         return false;
     }
-
+    
+    protected function getConfig()
+    {
+        $config = new ServicesConfig();
+        $config->secretApiKey = MODULE_PAYMENT_SECURESUBMIT_SECRET_API_KEY;
+        $config->serviceUrl = ($this->enableCryptoUrl) ?
+                              'https://cert.api2-c.heartlandportico.com/':
+                              'https://cert.api2.heartlandportico.com';
+        $service = new CreditService(
+                $config
+        );
+        return $service;
```

Comment:
It doesn't look like we are using the instantiated service above and are calling `authorize` and `charge` on the `CreditCardData` object. Let's update lines 232-235 to call `ServicesContainer::configure` and rename this method to `setConfig`.

Reply:
updated